### PR TITLE
[Plugin] WasmEdge-Tensorflow and WasmEdge-TensorflowLite plugin

### PR DESCRIPTION
**Pull request by q82419**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: plugins/wasmedge_image/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  set_target_properties(wasmedgePluginWasmEdgeImagePNG
+    PROPERTIES
+    IMPORTED_LOCATION ${wasmedge_image_libpng_SOURCE_DIR}/.libs/libpng16.a
+    INTERFACE_INCLUDE_DIRECTORIES ${wasmedge_image_libpng_SOURCE_DIR}
+  )
+endif()
+
+# Need zlib and boost.
+find_package(ZLIB REQUIRED)
+find_package(Boost 1.74.0)
```
Comment (hydai) [inline plugins/wasmedge_image/CMakeLists.txt:105]: Do we need the version to be `1.74.0`?

**Review by hydai — CHANGES_REQUESTED** [review]
Please use `"string"sv` in the string literals.

File: plugins/wasmedge_image/CMakeLists.txt (hunk 1 above)
Comment (q82419) [reply plugins/wasmedge_image/CMakeLists.txt:105]: At least.
This is for the version >= 1.74.0.

**Review by hydai — APPROVED** [review]
Finally, we can migrate the wasmedge-tensorflow as plugins

**Pull request merged**